test/parallel/test-http2-create-client-connect.js

```diff
@@ -22,8 +22,8 @@ const URL = url.URL;
       [`http://localhost:${port}`],
       [new URL(`http://localhost:${port}`)],
       [url.parse(`http://localhost:${port}`)],
-      [{ port: port }, { protocol: 'http:' }],
-      [{ port: port, hostname: '127.0.0.1' }, { protocol: 'http:' }]
+      [{ port }, { protocol: 'http:' }],
+      [{ port, hostname: '127.0.0.1' }, { protocol: 'http:' }]
```

Comment:
This is an exception for the sake of consistency with the line above. Feel free to request changes here (and anywhere else).